packages/api/src/modules/predicate/controller.ts

```diff
@@ -219,7 +213,8 @@ export class PredicateController {
 
       const tx_reserved_balances = predicate_txs.reduce(
         (accumulator, transaction: Transaction) => {
-          transaction.assets.forEach((asset: Asset) => {
+          // TODO: Adjust logic to get assets from transaction resume
```

Comment:
- remove `leftJoin('t.assets', 'a')` and selects `a.assetId` `a.amount`

-  transaction.assets.forEach((asset: IAsset) -> get outputs `(accumulator, {resume: {outputs}}: Transaction)`, add validations by type on reduce and increments values on BN format (same value type on  resume.output)